tools/deps.nim

```diff
@@ -28,8 +28,13 @@ proc cloneDependency*(destDirBase: string, url: string, commit = commitHead,
     # from failing
     execRetry fmt"git clone -q {url} {destDir2}"
   if isGitRepo(destDir):
-    execRetry fmt"git -C {destDir2} fetch -q"
-    exec fmt"git -C {destDir2} checkout -q {commit}"
+    let saveDir = getCurrentDir()
```

Comment:
optional nit: `oldDir`, save is a bit of a mis-nomer

Reply:
ok, done.